.github/workflows/trigger-breaking-change-alert.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
   trigger-notifier:
     if: contains(github.event.pull_request.labels.*.name, 'breaking')
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/breaking-change-alert.yaml@branch-24.12
+    uses: rapidsai/shared-workflows/.github/workflows/breaking-change-alert.yaml@branch-25.02
```

Comment:
As part of fixing conflicts, needed to rerun the update version script. It looks like this line wasn't updated previously. So it got picked up now. Just flagging this for awareness

Note this GHA workflow came from PR: https://github.com/rapidsai/kvikio/pull/536